vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -217,10 +217,33 @@ def _check_scheme_supported(self,
         else:
             return False
 
+    def _is_fp4a4_nvfp4(self, weight_quant: BaseModel, input_quant: BaseModel):
+
+        if weight_quant is None or input_quant is None:
+            return False
+
+        is_group_quant = (
+            weight_quant.strategy == QuantizationStrategy.GROUP.value)
```

Comment:
Should the input be group quant as well?

Reply:
Technically a new strategy - “tensor_group”